app/assets/stylesheets/home.scss

```diff
@@ -80,3 +78,8 @@
 .home-warning-footer-wrapper {
     padding-bottom: 40px;
 }
+
+.home-parties-list_disclaimer {
+    text-align: center;
```

Comment:
Text is not align. I was able to do it using `width: '600px', marginLeft: 'auto', marginRight: 'auto'`

Reply:
The disclaimer is aligned to the center.

![image](https://user-images.githubusercontent.com/1250918/65424186-ce108200-de02-11e9-97e5-22de5ffcbe15.png)